### ISSUE TITLE
Filter episodes popup cannot get screen reader focus

### ISSUE THREAD

**Issue by lohnn** [lohnn/oss_player#63]
When opening the "Filter episode" popup the focus remains behind the popup. We should make sure that the focus is set inside the popup and "tapping outside" will dismiss the popup, just as an alert dialog would.
